src/js/components/SchemaForm.js

```diff
@@ -252,10 +257,18 @@ class SchemaForm extends React.Component {
       );
     });
 
+    if (!this.state.renderGemini) {
```

Comment:
Could we get a comment about why this is necessary? Something like: 'Gemini measures the height/width which causes the animation to cancel'